src/ci/bin/testDockerDeadlock.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+# import in shellcheck / CI / IntelliJ compatible ways
+# shellcheck source=/dev/null
+source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
+
+cromwell::build::setup_common_environment
+
+if [[ "${CROMWELL_BUILD_PROVIDER}" == "${CROMWELL_BUILD_PROVIDER_TRAVIS}" ]]; then
+  # Upgrade docker-compose so that we get the correct exit codes
+  docker-compose -version
+  sudo rm /usr/local/bin/docker-compose
+  curl -L "https://github.com/docker/compose/releases/download/1.23.2/docker-compose-$(uname -s)-$(uname -m)" > docker-compose
+  chmod +x docker-compose
+  sudo mv docker-compose /usr/local/bin
+  docker-compose -version
+fi
+
+export TEST_CROMWELL_TAG=test-only-do-not-push
+
+docker image ls -q broadinstitute/cromwell:"${TEST_CROMWELL_TAG}" | grep . || \
+CROMWELL_SBT_DOCKER_TAGS="${TEST_CROMWELL_TAG}" sbt server/docker
+
+# Turn off exit-on-error temporarily, as we expect an error
+set +o errexit
+CROMWELL_TAG="${TEST_CROMWELL_TAG}" \
+docker-compose \
+  -f scripts/docker-compose-mysql/docker-compose-deadlock.yml \
+  up \
+  --scale cromwell=3 \
```

Comment:
Just out of curiosity, why scale to only 3?

Reply:
Shrug. 3 was what was in Thibault's original example, and worked well enough.